renderers/liquidcrystal/tcMenuLiquidCrystal.cpp

```diff
@@ -24,6 +24,7 @@ LiquidCrystalRenderer::LiquidCrystalRenderer(LiquidCrystal& lcd, uint8_t dimX, u
 	this->editChar = '=';
     this->drewTitleThisTime = false;
     this->titleRequired = true;
+    strcpy(this->title, applicationInfo.name);
```

Comment:
You'll need to use strncpy_P for this as the data will be in program memory on AVR and ESP. For example:

 strncpy_P(title, applicationInfo.name, sizeof(title));
 title[sizeof(title)-1] = 0; 

It's always safe because, on devices that don't have program memory, it just calls the regular version.